test/src/test/java/org/corfudb/runtime/BackupRestoreIT.java

```diff
@@ -0,0 +1,640 @@
+package org.corfudb.runtime;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.apache.commons.lang.RandomStringUtils;
+import org.corfudb.integration.AbstractIT;
+import org.corfudb.protocols.wireprotocol.Token;
+import org.corfudb.runtime.collections.*;
```

Comment:
Do we allow wildcard imports in tests?  If not, please do remove them

Reply:
Not sure if it is allowed. Just removed.